samples/detect.v1p3beta1.js

```diff
@@ -0,0 +1,175 @@
+/**
+ * Copyright 2018, Google, LLC
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+//CF: change service path, update helpers file to newest library when able.
+
+function localizeObjects(file) {
+  // [START vision object localization]
+  // Imports the Google Cloud client libraries
+  const vision = require('@google-cloud/vision').v1p3beta1;
+  const fs = require('fs');
+
+  /**
+   * TODO(developer): Uncomment the following line before running the sample.
+   */
+  // const fileName = 'Local image file, e.g. /path/to/image.png';
+
+  // Standard Endpoint
+  const client = new vision.ImageAnnotatorClient();
+  const request = {
+    image: {content: fs.readFileSync(file)},
+  };
+
+  client
+    .objectLocalization(request)
+    .then(results => {
+      console.log(JSON.stringify(results, null, '\t'));
+    })
+    .catch(err => {
+      console.error('ERROR:', err);
+    });
+  // [END Object Localization ]
+}
+
+function localizeObjectsGCS(uri) {
+  // [START vision object localization]
```

Comment:
Fix this region tag, it's probably something like `vision_object_localization_gcs` but I have no idea – copy from canonical